ledger/participant-state/src/main/scala/com/daml/ledger/participant/state/v1/package.scala

```diff
@@ -105,10 +105,10 @@ package object v1 {
     * absolute contract identifiers.
```

Comment:
Please refine the comments above as well (they still reference relative and absolute contract identifiers).

Reply:
done.